crates/apub_lib_derive/src/lib.rs

```diff
@@ -0,0 +1,102 @@
+use proc_macro2::TokenStream;
+use quote::quote;
+use syn::{parse_macro_input, Data, DeriveInput};
+
+#[proc_macro_derive(ActivityHandler)]
+pub fn derive_activity_handler(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
```

Comment:
Could you add some more description here, I'm not sure what this macro or this function is doing.

Reply:
Added a comment. A macro generates code at compilation time, in this case it generates `impl ActivityHandler for PersonInboxActivities` (or another enum).